seyiram/CryptoKoffee
# Refactor to function component

## Changes:
Changed from Class based component to Functional based component
Updated Smart Contract
Updated Smart Contract test cases

## How to test:
- Start your Ganache Instance and connect to metamask
- Clone
- Run truffle migrate --> Smart contract should be successfully deployed with Ganache Running
- Run truffle test --> All 11 test cases should pass
- cd to client folder
- Run npm start --> App should be started on localhost:3000 with Connect to metamask button
- Click on Button and should connect to metamask, open console and you should see the connected wallet address.
